tools/gvm-lsc-deb-creator

```diff
@@ -110,14 +88,6 @@ then
   exit 1
 fi
 
-#
-# Set up error handling
-#
-handle_error() {
-  echo "DEB package generation failed" >&2
-  exit 1
-}
-trap handle_error ERR
```

Comment:
What's the reason for removing the signal handler?

Reply:
Sorry, I didn't mean to remove those parts, just in a rush.  I will fix those into a new PR also, I have been reading Debian documentation and policy and I feel there is a slightly better solution to the issue of an error on re-installation based on Debian policy workflow, so I will prepare that and prepare a new PR.